core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/AiServicesProcessor.java

```diff
@@ -381,6 +382,12 @@ public void findDeclarativeServices(CombinedIndexBuildItem indexBuildItem,
 
             String imageModelName = chatModelName; // TODO: should we have a separate setting for this?
 
+            AnnotationInstance namedAnno = declarativeAiServiceClassInfo.annotation(NAMED);
+            Optional<String> beanName = Optional.empty();
```

Comment:
Is this the @Named annotation? We had an issue with the nonbinding aspect of that annotation in the past.

Reply:
It is yeah

Reply:
Yes, this is **@ Named** annotation. So the non binding aspect is that this annotation can't be implemented well in Quarkus ? Does it apply in this specific case ? Or maybe, it's just not recommended to use it ?

Reply:
It shouldn't be a problem in this case as it is entirely opt-in